pkg/scheduler/event_handler.go

```diff
@@ -312,3 +340,11 @@ func (s *Scheduler) deleteCluster(obj interface{}) {
 		s.schedulerEstimatorWorker.Add(cluster.Name)
 	}
 }
+
+func schedulerNameFilter(schedulerNameFromOptions, schedulerName string) bool {
+	if schedulerName == "" {
```

Comment:
Can you remind me when the `schedulerName` could be `empty`?

During the upgrade phase?

Reply:
Yeah, mainly for the upgrade case.